src/backends.jl

```diff
@@ -131,21 +132,44 @@ CurrentBackend(sym::Symbol) = CurrentBackend(sym, _backend_instance(sym))
 
 # ---------------------------------------------------------
 
-_fallback_default_backend() = backend(GRBackend())
-
-function _pick_default_backend()
-    if (env_default = get(ENV, "PLOTS_DEFAULT_BACKEND", "")) != ""
-        if (sym = Symbol(lowercase(env_default))) in _backends
-            backend(sym)
-        else
-            @warn """You have set PLOTS_DEFAULT_BACKEND=$env_default, but it is not a valid backend package.
-            Choose from: \n\t$(join(sort(_backends), "\n\t"))
-            """
-            _fallback_default_backend()
-        end
+function load_default_backend()
+    CURRENT_BACKEND.sym =
+        load_preferences(Plots, "default_backend", get(ENV, "PLOTS_DEFAULT_BACKEND", "gr")) |>
```

Comment:
This logic seems backwards. The environment variable should take precedence over Preferences.jl.

Reply:
Right.